src/routes/console/project-[project]/settings/migrations/+page.svelte

```diff
@@ -0,0 +1,284 @@
+                    <div class="u-margin-block-start-8">
+                        <Arrow direction="down" />
+                    </div>
+                    <div class="avatar u-margin-block-start-8" style="--size: {48 / 16}rem">
+                        <span class="icon-cloud" />
+                    </div>
+                    <Button class="u-margin-block-start-20" secondary on:click={openImportWizard}>
+                        Import data
+                    </Button>
+                </div>
+            {/if}
+        </svelte:fragment>
+    </CardGrid>
+    {#if isSelfHosted}
+        <CardGrid>
+            <Heading tag="h3" size="7">Deploy to Cloud</Heading>
+            <p class="text">Export data from your project to Appwrite Cloud</p>
+            <svelte:fragment slot="aside">
+                <div class="import-box">
+                    <div class="u-flex u-cross-center u-gap-8">
+                        <div class="avatar" style="--size: {48 / 16}rem">
+                            <span class="icon-server" />
+                        </div>
+                        <Arrow direction="right" />
+                        <div class="avatar" style="--size: {48 / 16}rem">
+                            <span class="icon-cloud" />
+                        </div>
+                    </div>
+                    <Button class="u-margin-block-start-48" secondary on:click={deployToCloud}
+                        >Deploy to Cloud</Button>
+                </div>
+            </svelte:fragment>
+        </CardGrid>
+    {:else}
+        <CardGrid>
+            <Heading tag="h3" size="7">Export to self-hosted instance</Heading>
+            <p class="text">Export data from your project to a self-hosted instance</p>
+            <svelte:fragment slot="aside">
+                <div class="import-box">
+                    <div class="u-flex u-cross-center u-gap-8">
+                        <div class="avatar" style="--size: {48 / 16}rem">
+                            <span class="icon-cloud" />
+                        </div>
+                        <Arrow direction="right" />
+                        <div class="avatar" style="--size: {48 / 16}rem">
+                            <span class="icon-server" />
+                        </div>
+                    </div>
+                    <Button
+                        class="u-margin-block-start-48"
+                        secondary
+                        on:click={() => (showExport = true)}>Export data</Button>
+                </div>
+            </svelte:fragment>
+        </CardGrid>
+    {/if}
+</Container>
+
+<ExportModal bind:show={showExport} />
+
+<Details bind:migrationId={details} migrations={data.migrations} />
+
+<style lang="scss">
+    .import-box {
+        display: grid;
+        place-items: center;
+
+        border: 1px dashed hsl(var(--color-border));
+        border-radius: 1rem;
+
+        height: 100%;
+
+        padding: 1.5rem;
+    }
+</style>
```

Comment:
Why not Pink?

Reply:
I'm overriding styles in some places, or adding new ones.